src/npc.cpp

```diff
@@ -104,6 +104,36 @@ npc::npc()
     last_updated = calendar::turn;
 }
 
+standard_npc::standard_npc( const std::string &name, const std::vector<itype_id> &clothing,
+                            int sk_lvl, int s_str, int s_dex, int s_int, int s_per )
+{
+    this->name = name;
```

Comment:
Oh, yes. It's a variable of the base class so you cant initialize it in an ancestor. I overlooked that.